File: langs/c_cplusplus/00_continue_or_break.cpp
#include <stdio.h>
#include <string.h>

int main () {
         
    for (int i = 0; i < 10; i++) {
        if (i == 3) {  printf("You are here!\n"); break; }        
        printf("i = %d\n", i);
    }

    printf("--------------\n");

    for (int j = 0; j < 10; j++) {
        if (j == 3) {  printf("I am here!\n"); continue; }        
        printf("j = %d\n", j);
    }
}

File: langs/c_cplusplus/00_continue_or_break.js
function solution(){
    let text = "";
    for (let i = 0; i < 10; i++) {
        if (i === 3) { 
        text = console.log("You are here ")
        break; 
        }
        text = "  " + i + " ";
        console.log(text)
    }

    console.log("--------------------")

    let str = "";
    for (let i = 0; i < 10; i++) {
        if (i === 3) { 
        str = console.log("I am here ")
        continue; 
        }
        str = "  " + i + " ";
        console.log(str)
    }
}

const sol = solution();